crates/kitsune_p2p/kitsune_p2p/Cargo.toml

```diff
@@ -44,7 +44,7 @@ thiserror = "1.0.22"
 tokio = { version = "1.27", features = ["full"] }
 tracing = "0.1"
 tokio-stream = "0.1"
-tx5 = { version = "=0.0.5-alpha", optional = true }
+tx5 = { version = "=0.0.6-alpha", optional = true }
```

Comment:
No workspace version here?

Reply:
It's the only occurrence in the project. Should that become a workspace dep too?